lib/lib-table-cluster/src/test/java/org/sagebionetworks/table/cluster/TableIndexDAOImplTest.java

```diff
@@ -1034,7 +1030,7 @@ public void testColumnInfoAndCardinality(){
 		assertTrue(info.hasIndex());
 		assertEquals("_C13_idx_", info.getIndexName());
 		assertEquals(MySqlColumnType.TINYINT, info.getType());
-		assertEquals(new Integer(1), info.getMaxSize());

```

Comment:
add test case where we have a VARCHAR MySqlType and assert that maxSize's value equals an expected value